packages/service-library/src/servicelib/long_running_tasks/server_long_running_manager.py

```diff
@@ -11,7 +11,7 @@
 from .task import TasksManager
 
 
-class BaseLongRunningManager(ABC):
+class ServerLongRunningManager(ABC):
```

Comment:
```suggestion
class LongRunningTasksManagerServer(ABC):
```
Does this still need to be called Server if you renamed the other one?

Reply:
no longer needs server in the name at this point